net-libs/neon/neon-0.32.2.ebuild

```diff
@@ -15,7 +15,8 @@ KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~riscv ~s390 ~
 IUSE="doc expat gnutls kerberos libproxy nls pkcs11 ssl static-libs zlib"
```

Comment:
Do we need static-libs?

Reply:
Grepping shows no reverse dependencies of `net-libs/neon[static-libs]`, so am dropping this.